### ISSUE TITLE
5.1 Build failing

### ISSUE THREAD

**Issue by jovinbm** [tripviss/docker-varnish#18]
Building the image without querystring module works perfectly but adding the module fails (only thing changed is v1.0.2 instead of v1.0.1). I have also tried v1.0.1 ending in the same result.

DockerFile
```
FROM tripviss/varnish:5.1

COPY default.vcl /usr/local/etc/varnish/
ENV VARNISH_MEMORY 1G
ENV VARNISH_PORT 1900
EXPOSE 1900

# Install Querystring Varnish module
ENV QUERYSTRING_VERSION 1.0.2
ENV QUERYSTRING_FILENAME libvmod-querystring-1.0.2.tar.gz
RUN set -xe \
    && curl -fSL "https://github.com/Dridi/libvmod-querystring/releases/download/v$QUERYSTRING_VERSION/vmod-querystring-$QUERYSTRING_VERSION.tar.gz" -o "$QUERYSTRING_FILENAME" \
    && mkdir -p /usr/local/src/libvmod-querystring \
    && tar -xzf "$QUERYSTRING_FILENAME" -C /usr/local/src/libvmod-querystring --strip-components=1 \
    && rm "$QUERYSTRING_FILENAME" \
    && cd /usr/local/src/libvmod-querystring \
    && gnuArch="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)" \
    && ./configure \
        --build="$gnuArch" \
        VARNISHSRC=/usr/local/src/varnish \
    && make -j "$(nproc)" \
    && make install \
    && rm -r /usr/local/src/libvmod-querystring
```
Error Screenshot
<img width="1920" alt="screen shot 2017-06-22 at 6 25 28 pm" src="https://user-images.githubusercontent.com/8378840/27443606-4f8ddd7e-577c-11e7-8df5-a02dd5c87098.png">

**Comment by jovinbm** [comment]
Also note that I've removed the line 
```
./autogen.sh
```
since `autogen.sh` is now not included in the build anymore at [https://github.com/Dridi/libvmod-querystring/releases](https://github.com/Dridi/libvmod-querystring/releases) > vmod-querystring-1.0.2.tar.gz

**Comment by teohhanhui** [comment]
I believe the instructions are not up-to-date. Please try adapting it to that of https://github.com/Dridi/libvmod-querystring/blob/v1.0.2/README.rst#installation instead.

If you continue to have issues, please open an issue in the issues tracker for libvmod-querystring. Otherwise, you can help us update the installation example by submitting a pull request.

**Comment by teohhanhui** [comment]
Fixed by #26
